From a227e7ad566c8b969b5a1848b5578dcd7c349a52 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Thu, 29 Aug 2019 22:32:55 +0000
Subject: [PATCH] joyent/bugview#37 illumos-live links should be rewritten to
 point at smartos-live Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved
 by: Joshua M. Clulow <jmc@joyent.com>

---
 jirapub.js | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/jirapub.js b/jirapub.js
index 740e206..9a8adee 100644
--- a/jirapub.js
+++ b/jirapub.js
@@ -569,6 +569,11 @@ fix_url(input)
 				m: '/smartos-live',
 				p: '/joyent/smartos-live'
 			},
+			{
+				h: 'github.com',
+				m: '/illumos-live',
+				p: '/joyent/smartos-live'
+			},
 			{
 				h: 'github.com',
 				m: '/illumos-extra',
-- 
2.21.0

